a.found the Typeerror message module not callable
b.searched it in google and found the solution
c.if you want to call other python file you have to write from file_name import file_name